LoopKitUI/Views/ScheduleEditor.swift

```diff
@@ -138,26 +141,43 @@ struct ScheduleEditor<Value: Equatable, ValueContent: View, ValuePicker: View, A
         }
     }
     
-    private var setupConfigurationPage: some View {
+    private var configurationPage: some View {
         switch mode {
         case .legacySettings:
-            return AnyView(wrappedPage)
-        case .acceptanceFlow, .settings:
-            return AnyView(configurationPage)
+            return AnyView(navigationPage)
+        case .acceptanceFlow:
+            return AnyView(page)
+        case .settings:
+            return AnyView(pageWithCancel)
         }
     }
     
-    private var wrappedPage: some View {
+    private var navigationPage: some View {
         NavigationView {
-            configurationPage
-            .navigationBarItems(
-                leading: cancelButton, // add in cancel button if modal
-                trailing: trailingNavigationItems
+            page
+                .navigationBarItems(
+                    leading: cancelButton, // add in cancel button if modal
+                    trailing: trailingNavigationItems
```

Comment:
[nit] spacing

Reply:
This is indented how Xcode wants it.  I could reduce the vertical space used...

Reply:
Gotcha - no need to reduce vertical space :)